src/components/NotFound/NotFound.scss

```diff
@@ -0,0 +1,34 @@
+@import "../../styles/utils/mixins";
+@import "../../styles/utils/variables";
+
+#main {
+  display: table;
+  width: 100%;
+  height: calc(100vh - 40px - 283px);
+  text-align: center;
+  background-color: #3B3E4A;
+
+  @include onTablet {
+    height: calc(100vh - 48px - 101px);
+  }
+
+  @include onDesktop {
+    height: calc(100vh - 64px - 101px);
+  }
+
+  font-family: "Lato", sans-serif;
+  color: #F1F2F9;
```

Comment:
use this color for text
![image](https://github.com/fe-may23-BugHunters/product_catalog_fe/assets/65409847/cb562810-bec4-4bd4-9ef3-c90d291a53d7)

Reply:
and create $color-secondary: #75767F; variable in file _variables.scss